core/field.js

```diff
@@ -232,7 +232,7 @@ Blockly.Field.prototype.getText_;
  *     clicked. Blockly will automatically set the field as clickable if this
  *     method is defined.
  * @return {void}
- * @protected
+ * @package
  */
 Blockly.Field.prototype.showEditor_;
```

Comment:
Remove the _

Reply:
I believe this one will have to fall under an exception as too many fields out there use this as public API, so changing it will break a lot of people.
I propose either: 
- a, doing the above. Leave the name as is, and change it to @package.
- b, Leave it as @protected, and add a wrapper. something like showEditor that is public and just calls showEditor_ internally.

Reply:
Ah, ok. It might be nice to have a wrapper so new people can use something that doesn't look private.

Reply:
Done.